src/cupbearer/scripts/conf/train_classifier_conf.py

```diff
@@ -1,21 +1,50 @@
+import dataclasses
 import os
 from dataclasses import dataclass
 from typing import Optional
 
-from cupbearer.data import DatasetConfig
+from cupbearer.data import DatasetConfig, NoData
 from cupbearer.models import CNN, MLP, ModelConfig
-from cupbearer.utils.config_groups import config_group
+from cupbearer.utils.config_groups import (
+    config_group,
+    register_config_group,
+)
 from cupbearer.utils.optimizers import Adam, OptimizerConfig
 from cupbearer.utils.scripts import DirConfig, ScriptConfig
-from simple_parsing.helpers import dict_field, mutable_field
+from cupbearer.utils.utils import BaseConfig
+from simple_parsing.helpers import mutable_field
+
+
+@dataclass(kw_only=True)
+class ValidationConfig(BaseConfig):
+    val: Optional[DatasetConfig] = config_group(DatasetConfig, NoData)
+    clean: Optional[DatasetConfig] = config_group(DatasetConfig, NoData)
+    backdoor: Optional[DatasetConfig] = config_group(DatasetConfig, NoData)
```

Comment:
As far as I can tell the names have no actual difference in what they actually imply besides convention.

On the other hand there are a lot of things you could wish for, but I think the best thing right now is just to have something good enough. As such having these names might be better than `val1`, `val2`, `val3`, ... as then the logs are more readable as long as you're following convention.

The following are the validation I've been wanting so far:
 - Regular validation dataset (same setting as training)
 - Clean, no backdoor, no adversarial attack, etc.
 - Backdoor, p_backdoor = 1.0 to see accuracy for backdoor detection
 - Alt. backdoor to use something like the backdoor but where backdoor label should not be used (only Wanet has this implemented right now, but I can see a use for at least corner as well)
---
For now I say this is good enough and we add any changes as needed.